Resources/config/services.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
You can inject the parameters, see http://symfony.com/doc/current/service_container/parameters.html for more information.